torch/distributed/fsdp/_shard_utils.py

```diff
@@ -124,3 +130,22 @@ def _create_chunk_sharded_tensor(
     return ShardedTensor._init_from_local_shards_and_global_metadata(
         local_shards, sharded_tensor_metadata=sharded_tensor_metadata, process_group=pg
     )
+
+
+def _create_chunk_dtensor(
```

Comment:
Is this API going to eventually be used in FSDP in a future diff?

Reply:
> Is this API going to eventually be used in FSDP in a future diff?

Ye. I am working on the follow up diffs. Should be pushed out soon.